integreat_cms/cms/views/external_calendars/external_calendar_form_view.py

```diff
@@ -95,6 +95,7 @@ def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
                 )
         else:
             external_calendar_form.instance.region = self.request.region
+            external_calendar_form.instance.created_by = request.user
```

Comment:
Should this happen on every update? Or just when the external calendar is created initially?

Reply:
In that case, we'll for sure also want to know *when* it was last changed